renovate.json

```diff
@@ -12,7 +12,7 @@
   "schedule": ["after 10pm every weekday", "before 4am every weekday", "every weekend"],
   "baseBranches": ["main"],
   "ignoreDeps": ["@types/node", "@types/selenium-webdriver", "selenium-webdriver"],
-  "packageFiles": ["WORKSPACE", "package.json"],
+  "packageFiles": ["WORKSPACE", "package.json", ".github/workflows/**/*.yml"],
```

Comment:
For this repository, our Github workflows should be local to the repository and shouldn't need updating correct?

Reply:
Missed that, you are right